app/views/content/life-as-a-teacher/explore-subjects/modern-languages.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Become a modern languages teacher
+title_paragraph: |-
+  <p>
+  By teaching pupils modern languages, you can help them gain a greater appreciation and understanding of the world. As they learn French, Spanish or German, your pupils will get a unique insight into the countries that speak these languages.</p>
```

Comment:
There are ITT courses in other languages such as Italian, Japanese, Mandarin and Russian

Should we mention these somewhere on the page (not necessarily here)?